designs/administrator/olm/sort-by-recently-updated/sort-by-recently-updated.md

```diff
@@ -0,0 +1,48 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Sort by Provider update on OperatorHub
```

Comment:
Should this major section be titled to include the tile view and list view option, too? Maybe something like "Sorting and Viewing Options in OperatorHub"?

Reply:
I really like that idea! I'll incorporate that! Thanks!